src/App.js

```diff
@@ -1,25 +1,84 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import { useState } from 'react';
+
+import contacts from './contacts.json';
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+    const fiveContacts = contacts.slice(0, 5);
+
+    const [addContact, setAddContact] = useState(fiveContacts);
+
+    const randomContact = c => {
+        const getRandomContact = c[Math.floor(Math.random() * c.length)];
+
+        return setAddContact([...addContact, getRandomContact]);
```

Comment:
Aca podriamos agregar una validacion para no agregar contactos ya existentes 👀 
```suggestion
 if (!addContact.includes(getRandomContact)) {
      return setAddContact([...addContact, getRandomContact]);
    }```